### DISCUSSION TITLE
Is it possible to use an underscore placeholder in some ways it's used in Scala ?

### DISCUSSION THREAD

**Discussion by PenghaiZhang** [gcanti/fp-ts#1935] [General]
Hi everybody, I am using both FP-TS and Scala in my project. 
I am curious whether we can use an underscore placeholder to make the code a bit more concise.

For example, in Scala an underscore placeholder in an expression is replaced by a anonymous parameter. So:

`_ + 1` for `x => x + 1`
`f(_)` for  `x => f(x)`

Any ideas guys ?

**Comment by samhh** [comment]
Currying can solve this with some caveats (keep things unary and non-optional). `x => f(x)` is equivalent to `f`, `x => x + 1` is equivalent to `add(1)`.
